templates/docs/examples/patterns/search-and-filter/combined.html

```diff
@@ -0,0 +1,17 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Search and filter / Combined{% endblock %}
+
+{% block standalone_css %}patterns_search-and-filter{% endblock %}
+
+{% block content %}
+{% with is_combined = true %}
+<section>{% include 'docs/examples/patterns/search-and-filter/chip-overflow.html' %}</section>
+<section>{% include 'docs/examples/patterns/search-and-filter/default.html' %}</section>
```

Comment:
Nit picking a bit, but could we have some order to the combined examples, at least having the default to be the first?

Reply:
@bartaz revised!